cpu/stm32f1/include/cpu_conf.h

```diff
@@ -52,7 +52,11 @@ extern "C" {
  * @brief   Flash page configuration
  * @{
  */
+#if defined(CPU_MODEL_STM32F103C8) || defined(CPU_MODEL_STM32F103CB) || defined(CPU_MODEL_STM32F103RB)
+#define FLASHPAGE_SIZE      (1024U)
```

Comment:
don't we need to fix the `FLASHPAGE_NUMOF` for those CPU variants below then, too? That is `64U` for STM32F103C8 and `128U) for STM32F103CB and STM32F103RB.

Reply:
Indeed, I overlooked this and fixed it now. Thanks for being watchful.